hummingbot/connector/exchange/bittrex/bittrex_utils.py

```diff
@@ -1,12 +1,17 @@
+from decimal import Decimal
+
 from hummingbot.client.config.config_var import ConfigVar
 from hummingbot.client.config.config_methods import using_exchange
-
+from hummingbot.core.data_type.trade_fee import TradeFeeSchema
 
 CENTRALIZED = True
 
 EXAMPLE_PAIR = "ZRX-ETH"
 
-DEFAULT_FEES = [0.25, 0.25]
+DEFAULT_FEES = TradeFeeSchema(
+    maker_percent_fee_decimal=Decimal("0.0035"),
+    taker_percent_fee_decimal=Decimal("0.0035"),
+)
```

Comment:
Is it correct to go from 0.25 to 0.35?

Reply:
I got the information from [here](https://bittrex.zendesk.com/hc/en-us/articles/115000199651-Bittrex-fees). I am defaulting to the lowest tier